### PR TITLE
Set values in Presenter

### PR DESCRIPTION
**Pull request by KyleQianliMa**
[EWM 8566](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=8566)

# Short description of the changes:

# Long description of the changes:

# Check list for the pull request
- [ ] I have read the [CONTRIBUTING]
- [ ] I have read the [CODE_OF_CONDUCT]
- [ ] I have added tests for my changes
- [ ] I have updated the documentation accordingly

# Check list for the reviewer
- [ ] I have read the [CONTRIBUTING]
- [ ] I have verified the proposed changes
- [ ] best software practices
    + [ ] all internal functions have an underbar, as is python standard
    + [ ] clearly named variables (better to be verbose in variable names)
    + [ ] code comments explaining the intent of code blocks
- [ ] All the tests are passing
- [ ] The documentation is up to date
- [ ] code comments added when explaining intent

# Manual test for the reviewer

# References

### CODE REVIEW

File: src/hyspecppt/hppt/experiment_settings.py [hunk 1]
```diff
@@ -16,5 +16,5 @@
 
 # default parameters
 DEFAULT_LATTICE = dict(a=1, b=1, c=1, alpha=90, beta=90, gamma=90, h=0, k=0, l=0)
-DEFAULT_EXPERIMENT = dict(Ei=20, s2=30, alpha_p=0, plot_type=PLOT_TYPES[1])
+DEFAULT_EXPERIMENT = dict(Ei=20, s2=30, alpha_p=0, plot_type=PLOT_TYPES[0])
```
Comment (AndreiSavici) [inline src/hyspecppt/hppt/experiment_settings.py]: Leave it at 1

File: src/hyspecppt/hppt/hppt_presenter.py [hunk 2]
```diff
@@ -8,6 +10,11 @@ def __init__(self, view, model):
         """Constructor"""
```
Comment (AndreiSavici) [inline src/hyspecppt/hppt/hppt_presenter.py]: Types and documentation

File: src/hyspecppt/hppt/hppt_presenter.py [hunk 3]
```diff
@@ -8,6 +10,11 @@ def __init__(self, view, model):
...
         self._view = view
         self._model = model
+        self.view.SCW.set_values(DEFAULT_LATTICE)
+        self.view.EW.initializeCombo(PLOT_TYPES)
+        self.view.EW.set_values(DEFAULT_EXPERIMENT)
+        self.view.CW.set_values(DEFAULT_CROSSHAIR)
+        self.view.SelW.set_SC_toggle(True)
 
     @property
     def view(self):
```
Comment (AndreiSavici) [inline src/hyspecppt/hppt/hppt_presenter.py:30]: Documentation and types

File: src/hyspecppt/hppt/hppt_view.py [hunk 4]
```diff
@@ -113,6 +109,13 @@ def __init__(self, parent: Optional["QObject"] = None) -> None:
         selector_layout.addWidget(self.sc_rb)
         self.setLayout(selector_layout)
 
+    def set_SC_toggle(self, toggle: bool) -> None:
+        """Sets widget display based on the values dictionary
+        Args:
+            Bool
```
Comment (AndreiSavici) [inline src/hyspecppt/hppt/hppt_view.py]: What does it mean for True/False

File: src/hyspecppt/hppt/hppt_view.py [hunk 5]
```diff
@@ -271,6 +274,18 @@ def validate_inputs(self, *dummy_args, **dummy_kwargs) -> None:
         print(self.sender)
         print("args: ", dummy_args, " kwargs: ", dummy_kwargs)
 
+    def set_values(self, values: dict[str, float]) -> None:
+        """Sets widget display based on the values dictionary
+
+        Args:
+            values (dict): a dictionary that contains
+            Ei, S2, alpha_p, plot_types values
```
Comment (AndreiSavici) [inline src/hyspecppt/hppt/hppt_view.py:283]: Can you check in the entire source code for S2 vs s2. I don't care which, but let's be consistent

**Review by AndreiSavici — CHANGES_REQUESTED** [review]
Is there a test for the presenter?

**Comment by KyleQianliMa** [comment]
@AndreiSavici I saw in Shiver there is no tests for presenter. Only three folders Data, View, Model.

File: src/hyspecppt/hppt/hppt_view.py (hunk 5 above)
Comment (KyleQianliMa) [reply src/hyspecppt/hppt/hppt_view.py:283]: Setting all to upper S2.

**Review by AndreiSavici — APPROVED** [review]

**Pull request merged**